docs/installation.rst

```diff
@@ -116,5 +116,43 @@ You could also do this using Django's `manage.py shell`:
     ... )
     <ValidationService: default>
 
+.. _webauthn-setup:
+
+WebAuthn Setup
+--------------
+
+In order to support WebAuthn_ devices, you have to install the py_webauthn_ package.
+It's a ``django-two-factor-auth`` extra so you can select it at install time:
+
+.. code-block:: console
+
+    $ pip install django-two-factor-auth[webauthn]
+
+You need to include the plugin in your Django settings:
+
+.. code-block:: python
+
+    INSTALLED_APPS = (
+        ...
+        'two_factor.plugins.webauthn',
+    )
```

Comment:
All the other codeblocks in the same file were doing INSTALLED_APPS as a tuple. It's fine by me if INSTALLED_APPS is a list but in that case all of them should be, so that's what I did.